test/apps/iodemo/io_demo.cc

```diff
@@ -448,6 +450,8 @@ class P2pDemoCommon : public UcxContext {
             case UCS_MEMORY_TYPE_HOST:
                 free(_buffer);
                 break;
+            case UCS_MEMORY_TYPE_UNKNOWN:
+                break;
```

Comment:
I knew it was a little confused. Let me explain. The idea was to support another type of Buffer where memory allocation/free is not needed. For this type Buffer, memory allocation/free will be handled by UCX instead of upper application. UCS_MEMORY_TYPE_UNKNOWN is used to indicate this type of Buffer. I knew it was not a good representation. Any better idea? BTW I have thought of introducing a new class. But avoiding using Buffer would have a huge impact on currect code structure because currectly Callback<->BufferIov<->Buffer relationship is very close.

Reply:
ok, ic